jerry-core/debugger/debugger-ws.c

```diff
@@ -291,14 +293,14 @@ jerry_process_handshake (int client_socket, /**< client socket */
 
   text_p = "HTTP/1.1 101 Switching Protocols\r\nUpgrade: websocket\r\nConnection: Upgrade\r\nSec-WebSocket-Accept: ";
 
-  if (!jerry_debugger_send_tcp ((const uint8_t *) text_p, strlen (text_p))
-      || !jerry_debugger_send_tcp (request_buffer_p + sha1_length + 1, 27))
+  if (!jerry_debugger_send_transport ((const uint8_t *) text_p, strlen (text_p))
+      || !jerry_debugger_send_transport (request_buffer_p + sha1_length + 1, 27))
```

Comment:
This is quite websocket specific. As far as I know websockets is available only through normal sockets.

Reply:
i could leave it with just  _tcp if it's ok, @martijnthe ?

Reply:
> As far as I know websockets is available only through normal sockets.

At this point I'm regarding the websocket framing and handshaking to be part of the JrS debugger protocol as you yourself suggested a year ago: https://github.com/jerryscript-project/jerryscript/pull/1557#issuecomment-277245868

Reply:
It was a long time ago when I worked on this. But it seems a reasonable idea. Probably the handshake could be an optional feature.